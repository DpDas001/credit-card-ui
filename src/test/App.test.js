import React from 'react';
import { shallow, mount } from '../enzyme';
import App from '../App';



describe("should rendered Main", () => {
  it("should render my Main", () => {
    const wrapper = shallow(<App />);
    expect(wrapper.getElements()).toMatchSnapshot();
  });
});

