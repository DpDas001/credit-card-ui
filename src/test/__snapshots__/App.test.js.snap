// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should rendered Main should render my Main 1`] = `
Array [
  <div
    className="App"
  >
    <Main />
  </div>,
]
`;
